typescript/infra/scripts/check/check-owner-ica.ts

```diff
@@ -39,7 +33,9 @@ async function main() {
 
   const checkOwnerIcaChains = (
     chains?.length ? chains : Object.keys(icas)
-  ).filter(isEthereumProtocolChain);
+  ).filter(
+    (chain) => isEthereumProtocolChain(chain) && !chainsToSkip.includes(chain),
```

Comment:
what's the purpose of `chainsToSkip`? iiuc we want to skip the chains for some things and not for other things, a comment of how it's intended to be used where it's defined would be nice. Or maybe to have it in the name

Reply:
added a comment in chain.ts it's defined